src/modules/home/components/YourTripsSection.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, Flex, Text } from "@chakra-ui/react";
+
+import GoogleMapsCard from "@/modules/components/GoogleMapsCard";
+import { DirectionsServiceProps } from "@react-google-maps/api";
+import { FaMapMarkerAlt } from "react-icons/fa";
+import { MdModeStandby } from "react-icons/md";
+import { RxDividerVertical } from "react-icons/rx";
+
+//request data
+const data = {
+  origin: "新北市政府",
+  waypoints: [
+    {
+      location: "中正紀念堂",
+    },
+    {
+      location: "國父紀念館",
+    },
+  ],
+  destination: "台北市政府",
+  travelMode: "DRIVING" as DirectionsServiceProps["options"]["travelMode"],
+  time: "18:00",
+  date: "2023-05-23",
+};
+//parse data for GoogleMapsCard
+const data2 = {
+  origin: data.origin,
+  waypoints: data.waypoints,
+  destination: data.destination,
+  travelMode: data.travelMode,
+};
```

Comment:
`data2` might not be a good naming decision.
Maybe name it `gMapData` will make it much more readable 😊
```suggestion
const gMapData = {
  origin: data.origin,
  waypoints: data.waypoints,
  destination: data.destination,
  travelMode: data.travelMode,
};
```